datafusion/physical-expr/src/aggregate/count_distinct.rs

```diff
@@ -31,35 +31,30 @@ use datafusion_common::{DataFusionError, Result};
 use datafusion_expr::Accumulator;
 
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
-struct DistinctScalarValues(Vec<ScalarValue>);
+struct DistinctScalarValues(ScalarValue);
```

Comment:
We could use `HashSet<DistinctScalarValues, RandomState>` instead and remove this struct.